quinn-proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quinn-proto"
-version = "0.8.4"
+version = "0.8.5"
```

Comment:
No need to bump -proto since you've made no changes there.

Reply:
Gotcha, fixed